hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeysDeleteRequest.java

```diff
@@ -123,13 +127,15 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager, TermIn
             .setVolumeName(volumeName).setBucketName(bucketName);
 
     boolean deleteStatus = true;
+    long startNanos = Time.monotonicNowNanos();
+    String finalVolumeName = volumeName;
+    String finalBucketName = bucketName;
     try {
-      ResolvedBucket bucket =
-          ozoneManager.resolveBucketLink(Pair.of(volumeName, bucketName), this);
+      ResolvedBucket bucket = captureLatencyNs(perfMetrics.getDeleteKeysResolveBucketLatencyNs(), () ->
+          ozoneManager.resolveBucketLink(Pair.of(finalVolumeName, finalBucketName), this));
       bucket.audit(auditMap);
       volumeName = bucket.realVolume();
       bucketName = bucket.realBucket();
-
       mergeOmLockDetails(omMetadataManager.getLock()
```

Comment:
Minor nit: original new line
```suggestion
      
      mergeOmLockDetails(omMetadataManager.getLock()
```